### PR TITLE
[OBSDEF-12244][OBSDEF-11631][OBSDEF-11492][OBSDEF-11616] Event rules for OBJSOP- and OBJSC-SP- issues' refactor

### PR DESCRIPTION
**Pull request by kirillston**
## Purpose
[OBSDEF-12244](https://jira.cec.lab.emc.com/browse/OBSDEF-12244)
Added component name to all remedies related to ObjScale- and SP-Operator events

[OBSDEF-11631](https://jira.cec.lab.emc.com/browse/OBSDEF-11631)
Recovery SP-Operator event rules added

[OBSDEF-11492](https://jira.cec.lab.emc.com/browse/OBSDEF-11492)
SP-Operator event rules are specified for ObjectStore components

[OBSDEF-11616](https://jira.cec.lab.emc.com/browse/OBSDEF-11616)
CSR and TLS related issues are specified for ObjectStore

## PR checklist
- [ ] make test passed
- [x] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
_Paste URL of charts-custom-ci job here_

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: ecs-cluster/alerts/remedies/OBJSOP-1000_DISK_REPLACEMENT_remedies.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSOP-1000
+  component: objectstore
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/remedies/OBJSOP-1000_DISK_REPLACEMENT_remedies.yaml:2]: if the operator is generating the event, it should;d be operator name. We want to make sure that we can find out the person based on the component who is responsible to the generate events. This event is generated on the objectstore but it is operator or sp-operator who is generating it.

File: ecs-cluster/alerts/remedies/OBJSOP-1001_PMM_remedies.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSOP-1001
+  component: objectstore
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/remedies/OBJSOP-1001_PMM_remedies.yaml:2]: same

File: ecs-cluster/alerts/remedies/OBJSOP-1002_TMM_remedies.yaml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSOP-1002
+  component: objectstore
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/remedies/OBJSOP-1002_TMM_remedies.yaml:2]: same

File: ecs-cluster/alerts/remedies/OBJSOP-1003_UPGRADE_remedies.yaml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSOP-1003
+  component: objectstore
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/remedies/OBJSOP-1003_UPGRADE_remedies.yaml:2]: same

File: ecs-cluster/alerts/remedies/OBJSOP-1004_HORIZONTAL_EXPAND_remedies.yaml [hunk 5]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSOP-1004
+  component: objectstore
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/remedies/OBJSOP-1004_HORIZONTAL_EXPAND_remedies.yaml:2]: same

File: ecs-cluster/alerts/remedies/OBJSOP-1004_VERTICAL_EXPAND_remedies.yaml [hunk 6]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSOP-1005
+  component: objectstore
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/remedies/OBJSOP-1004_VERTICAL_EXPAND_remedies.yaml:2]: same

File: objectscale-manager/alerts/remedies/OBJSC-MGR-HC-1000_PRE_UPDATE_remedies.yaml [hunk 7]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSC-MGR-HC-1000
+  component: objectscale-manager
```
Comment (vgupta-mickey) [inline objectscale-manager/alerts/remedies/OBJSC-MGR-HC-1000_PRE_UPDATE_remedies.yaml:2]: I would say...you need to use specific component name - objectscale-operator or objectscale-sp-operator depending on who is generating it.

File: objectscale-manager/alerts/remedies/OBJSC-SP-0001_DISK_REPLACEMENT_remedies.yaml [hunk 8]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSC-SP-0001
+  component: objectscale-manager
```
Comment (vgupta-mickey) [inline objectscale-manager/alerts/remedies/OBJSC-SP-0001_DISK_REPLACEMENT_remedies.yaml:2]: same

File: objectscale-manager/alerts/remedies/OBJSC-SP-0002_PMM_remedies.yaml [hunk 9]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSC-SP-0002
+  component: objectscale-manager
```
Comment (vgupta-mickey) [inline objectscale-manager/alerts/remedies/OBJSC-SP-0002_PMM_remedies.yaml:2]: same

File: objectscale-manager/alerts/remedies/OBJSC-SP-0003_TMM_remedies.yaml [hunk 10]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSC-SP-0003
+  component: objectscale-manager
```
Comment (vgupta-mickey) [inline objectscale-manager/alerts/remedies/OBJSC-SP-0003_TMM_remedies.yaml:2]: same

File: objectscale-manager/alerts/remedies/OBJSOP-2001-CSR_NOTIFICATIONS_remedies.yaml [hunk 11]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSOP-2001
+  component: objectscale-manager
```
Comment (vgupta-mickey) [inline objectscale-manager/alerts/remedies/OBJSOP-2001-CSR_NOTIFICATIONS_remedies.yaml:2]: same

File: objectscale-manager/alerts/remedies/OBJSOP-2002-TLS_CERT_EXPIRY_remedies.yaml [hunk 12]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJSOP-2002
+  component: objectscale-manager
```
Comment (vgupta-mickey) [inline objectscale-manager/alerts/remedies/OBJSOP-2002-TLS_CERT_EXPIRY_remedies.yaml:2]: same

**Review by vgupta-mickey — CHANGES_REQUESTED** [review]

File: ecs-cluster/alerts/remedies/OBJSOP-1000_DISK_REPLACEMENT_remedies.yaml (hunk 1 above)
Comment (kirillston) [reply ecs-cluster/alerts/remedies/OBJSOP-1000_DISK_REPLACEMENT_remedies.yaml]: Thank you for explanation, Vinod!
Changes are done

File: ecs-cluster/alerts/remedies/OBJSOP-1001_PMM_remedies.yaml (hunk 2 above)
Comment (kirillston) [reply ecs-cluster/alerts/remedies/OBJSOP-1001_PMM_remedies.yaml]: Done!

File: ecs-cluster/alerts/remedies/OBJSOP-1002_TMM_remedies.yaml (hunk 3 above)
Comment (kirillston) [reply ecs-cluster/alerts/remedies/OBJSOP-1002_TMM_remedies.yaml]: Done!

File: ecs-cluster/alerts/remedies/OBJSOP-1003_UPGRADE_remedies.yaml (hunk 4 above)
Comment (kirillston) [reply ecs-cluster/alerts/remedies/OBJSOP-1003_UPGRADE_remedies.yaml]: Done!

File: ecs-cluster/alerts/remedies/OBJSOP-1004_HORIZONTAL_EXPAND_remedies.yaml (hunk 5 above)
Comment (kirillston) [reply ecs-cluster/alerts/remedies/OBJSOP-1004_HORIZONTAL_EXPAND_remedies.yaml]: Done!

File: ecs-cluster/alerts/remedies/OBJSOP-1004_VERTICAL_EXPAND_remedies.yaml (hunk 6 above)
Comment (kirillston) [reply ecs-cluster/alerts/remedies/OBJSOP-1004_VERTICAL_EXPAND_remedies.yaml]: Done!

File: objectscale-manager/alerts/remedies/OBJSC-MGR-HC-1000_PRE_UPDATE_remedies.yaml (hunk 7 above)
Comment (kirillston) [reply objectscale-manager/alerts/remedies/OBJSC-MGR-HC-1000_PRE_UPDATE_remedies.yaml]: Done!

File: objectscale-manager/alerts/remedies/OBJSC-SP-0002_PMM_remedies.yaml (hunk 9 above)
Comment (kirillston) [reply objectscale-manager/alerts/remedies/OBJSC-SP-0002_PMM_remedies.yaml]: Done!

File: objectscale-manager/alerts/remedies/OBJSC-SP-0003_TMM_remedies.yaml (hunk 10 above)
Comment (kirillston) [reply objectscale-manager/alerts/remedies/OBJSC-SP-0003_TMM_remedies.yaml]: Done!

File: objectscale-manager/alerts/remedies/OBJSOP-2001-CSR_NOTIFICATIONS_remedies.yaml (hunk 11 above)
Comment (kirillston) [reply objectscale-manager/alerts/remedies/OBJSOP-2001-CSR_NOTIFICATIONS_remedies.yaml]: Done!

File: objectscale-manager/alerts/remedies/OBJSOP-2002-TLS_CERT_EXPIRY_remedies.yaml (hunk 12 above)
Comment (kirillston) [reply objectscale-manager/alerts/remedies/OBJSOP-2002-TLS_CERT_EXPIRY_remedies.yaml]: Done!

**Review by vgupta-mickey — APPROVED** [review]

**Review by BassinD — APPROVED** [review]

File: ecs-cluster/alerts/remedies/OBJSC-SP-0000_RECOVERY_remedies.yaml [hunk 13]
```diff
@@ -0,0 +1,5 @@
+- symptomid: OBJSC-SP-0000
+  component: objectscale-sp-operator
```
Comment (eddiepavkovic) [inline ecs-cluster/alerts/remedies/OBJSC-SP-0000_RECOVERY_remedies.yaml:2]: i think this now needs to be the jira component.

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]

File: ecs-cluster/alerts/remedies/OBJSC-SP-0000_RECOVERY_remedies.yaml (hunk 13 above)
Comment (antonbassin) [reply ecs-cluster/alerts/remedies/OBJSC-SP-0000_RECOVERY_remedies.yaml:2]: Jira components are not correct in general. do we have a kahm component for example?
operator and sp operator have different owners. 
Leave it as is, we will add the sp-operator jira component if needed.

Comment (eddiepavkovic) [reply ecs-cluster/alerts/remedies/OBJSC-SP-0000_RECOVERY_remedies.yaml:2]: The ask from mgmt is to use Jira components.  So that there's clear ownership

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]

**Pull request closed without merging**